packaging/zeek-agent.cfg.template

```diff
@@ -1 +1,41 @@
-# TODO
+# Configuration file for ZeekAgent. Most of these options should work with their
```

Comment:
I'd generally prefer not to change any defaults through the config template. If we want defaults to be different, we can do that in the code. With that said, anything here that Windows requires to be different than our built-in defaults?

Reply:
Nothing different required. I was mostly just adding the values into the file so that we could document what they were right there with the config. I updated the missing text with the comments from `configuration.h`, except the `groups` option since I'm still not sure what it does. I also fixed the existing options to be commented out, and to have the current default from the code.